test/src/test/java/org/corfudb/runtime/clients/LogUnitHandlerTest.java

```diff
@@ -118,6 +118,17 @@ public void canReadWrite()
                 .isEqualTo(testString);
     }
 
+    @Test
+    public void canGetTrimMarkAndLogSize()
```

Comment:
can we also add a test where the end address exceeds the segment end address when requesting a segment size?